assets/js/dependencies/async_loader.js

```diff
@@ -166,17 +166,17 @@
         else{
           url = url.replace("../","");
           if(url.charAt(0)!="/"){
-            url="/"+url;
+            url ="/"+url;
           }
           var base = $('base').attr('href');
           if(url.indexOf('?')>-1){
-            url = url + '&rstudio_layout=1&viewer_pane=1&RS_SHARED_SECRET=' + urlParam("RS_SHARED_SECRET")+"&Rstudio_port=" + urlParam("Rstudio_port");
```

Comment:
When you don't declare your variable using `var` you create a global variable. 
Just declare `var urlWParams` before the condition.
Won't it be easier to just add `?` to url if `url.indexOf('?')>-1` instead of duplicating the code that adds the parameters.